### PR TITLE
Formal Captain Update

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Formalizes the captain a bit---sprites and such from TG

- Gives the captain a formal parade uniform
- Gives the captain formal parade armor
- Gives the captain a rapier
 - Rapiers have a chance to block non-projectiles and have high armor penetration
- Gives the captain a pair of dress shoes
- Gives the captain a fancy crown 
  - Can make basic crowns out of gold; they have a tiny bit of armor on them.

Pics:

img1
img2
img3
img4

:cl: Fox McCloud
add: Adds in formal captain's uniform and armor
add: Adds rapier to captain's locker
add: Adds lace up shoes to captain's locker
add: Adds crown to captain's locker
/:cl:

### CODE REVIEW

**Comment by chopchop1614** [comment]
Holy shit I am all for this! Love it!

**Comment by Plotron** [comment]
DRESS SHOES WHEN???

**Comment by Jaywings** [comment]
Yes.

**Comment by FreeStylaLT** [comment]
good lord when will we get thrones and crowns at this point

**Comment by Fox-McCloud** [comment]
>good lord when will we get thrones and crowns at this point

Lemme add that too, real quick...

**Comment by MarsM0nd** [comment]
Might also want to make this a traitor target, and consider the SoP use of it. Are they allowed to wear it at all times, or only for a ceremony or like the antique gun.

**Comment by chopchop1614** [comment]
I would say at all times, just cause I want to wear it :P But in all seriousness maybe only for special events (that really never happen).

**Comment by Regen1** [comment]
What about a throne instead of the command console

And a scepter, captains scepter

And a crown

And instead of ID's we use rings

And the throne only works for the captain when he is wearing his ring and crown

And you have to be crowned baro- i mean kin- i mean captain before you can use the command throne

It's not lifeweb inspired I promise

**Comment by Jaywings** [comment]
How about we just limit the weapon with the current server rules. AKA. No randomly stabbing people. Get out of here @Regen1

**Comment by taukausanake** [comment]
I just want thrones construct-able with gold and detailed with diamonds afterwards.

And now I have to decide between the laser gun and a sword to carry as captain. Great.

**Comment by Fox-McCloud** [comment]
>And now I have to decide between the laser gun and a sword to carry as captain. Great.

You can carry the rapier in its sheath on your belt and wear your energy gun on your armor.

**Comment by FlattestGuitar** [comment]
This does need dress shoes added.

**Comment by Regen1** [comment]
In all seriousness though

I am :-1: :100: on the rapier

If it's a ceremonial one? Sure, but they are very rarely sharp, let alone combat efficient

Make it purely cosmetic and we're Gucci

**Comment by Fox-McCloud** [comment]
Alrighty, gave the Captain a pair of lace up shoes and added in the crowns.

**Comment by FreeStylaLT** [comment]
>added in the crowns.

>no pics

:eyes:

**Comment by chopchop1614** [comment]
This is awesome, all will bow to the king! Now for a dictator gamemode ;P

**Comment by Fox-McCloud** [comment]
>no pics

Scrub, look at the OP you nerdddd

**Comment by Spacemanspark** [comment]
As un-amused as I am with the passive aggresiveness, I don't see any issue with the rapier.

**Comment by Earthdivine** [comment]
Only thing I have a bit of an issue with is the blocking. It looks good with the set up and bearly does any damage

**Comment by chopchop1614** [comment]
It would be cool if you could use the rapier to like leap forward a tile or two and strike if there is an enemy there, kinda like what xenos do.

**Comment by Fox-McCloud** [comment]
>Only thing I have a bit of an issue with is the blocking. It looks good with the set up and bearly does any damage

It's basically a way crummier Riot Shield (for blocking) that can't be collapse and doesn't even block ranged projectiles.

**Comment by Bram961** [comment]
The block chance on that rapier might make this rather overpowered at 50 for its chance. It is also completely incorrect as rapiers are thrusting weapons which were (for good reason) commonly wielded with a parrying dagger.

**Comment by DaveTheHeadcrab** [comment]
We need a parrying dagger.
And a clown alternative, the partying dagger.

**Comment by taukausanake** [comment]
> Now for a dictator gamemode

Yes that sounds amazing but I wonder how it would be designed. Wouldn't be too easy

**Comment by Spacemanspark** [comment]
Dictator gamemode sounds like it'd be similar to the old mutiny gamemode we had at one point.

**Comment by sasanek12** [comment]
Jester,Cerbs and Meister when?

**Comment by Regen1** [comment]
Needs more Censor, character stats, skills and grimdark

**Comment by IcyV** [comment]
Not really feeling the block chance on the rapier, but I'm all for everything else.

**Comment by Necaladun** [comment]
o7 Oh Captain My Captain.

**Comment by BiberDark** [comment]
Very nice !

**Comment by KingPhilipIII** [comment]
I'm all for stealing the Captain's Rapier as an objective. As an abductor I can proudly declare myself Captain for stealing their leader's head ornamentation. Bleep Bloop Blop.

File: code/game/objects/items/weapons/melee/misc.dm [hunk 1]
```diff
@@ -20,6 +20,28 @@
 		to_chat(viewers(user), "<span class='suicide'>[user] is strangling \himself with the [src.name]! It looks like \he's trying to commit suicide.</span>")
 		return (OXYLOSS)
 
+/obj/item/weapon/melee/rapier
+	name = "captain's rapier"
+	desc = "An elegant weapon, for a more civilized age."
```
Comment (TheDZD) [inline code/game/objects/items/weapons/melee/misc.dm]: :eyes:

**Comment by Doukhan** [comment]
Rapier 5/10 brute damage?

It's nice to have TEH POWAH OF AUTHORITAH made flesh but this is a pretty big buff, it's not like he can't just get himself a butcher's cleaver anyway if he wanted to but still.

**Comment by KasparoVy** [comment]
Necaladun said it hahaha

Oh Captain, my Captain!

But Captain to kill all them!
Yes. Kill all them.

**Comment by KasparoVy** [comment]
@Fox-McCloud I whipped up some Vox versions of the sprites, pretty sure there won't be any code changes necessary as the Captain's carapace and all uniforms are already Vox species_fitted

_Vox Captain Formal Uniform_
voxcaptainformaluniform

_Vox Captain Formal Jacket_
voxcaptainformaljacket

Just gotta replace uniform.dmi and suit.dmi in icons/mob/species/vox with the files in this zip: 
[vox.zip](https://github.com/ParadiseSS13/Paradise/files/410529/vox.zip)

**Comment by Fox-McCloud** [comment]
Added in the Vox fitted versions.

**Pull request merged**